tensorflow/core/kernels/svd_op_gpu.cu.cc

```diff
@@ -93,6 +103,50 @@ class SvdOpGpu : public AsyncOpKernel {
           context, solver.Gesvd(jobu, jobvt, m, n, input, lda, outputS, outputU,
                                 ldu, outputVT, ldvt, dev_info_ptr + i),
           done);
+
+      // This is a bug in cuSolver:
+      // If n is one, then outputVT only contains zeros instead of ones.
+      // Hence, I need to fill outputVT manually
+      if (compute_uv_ && n==1) {
+        int64 count =  n * (full_matrices_ ? n : p);
+        const GPUDevice& d = context->eigen_device<GPUDevice>();
+        CudaLaunchConfig cfg = GetCudaLaunchConfig(count, d);
+        FillWithOnesKernel <<<cfg.block_count, 
```

Comment:
We already have a functor like this: https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/fill_functor.h#L63